web/client/components/mapcontrols/annotations/SelectAnnotationsFile.jsx

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
There are some strange style effect if you try to drop the file
initially it appears a div with red background 
![image](https://user-images.githubusercontent.com/11991428/36591952-09925ef6-1894-11e8-9cbf-5f3f9e47eb34.png)

then for a quick moment the dropzone moves down 
![image](https://user-images.githubusercontent.com/11991428/36591961-105ad57e-1894-11e8-8c4c-51297d6a8c58.png)

can you check this behaviour?

Reply:
It's a dropzone effect in windows that doesn't have the correct file type. We can remove the dropzone check if you prefer.